### ISSUE TITLE
Filter out missions by ID or NAME

### ISSUE THREAD

**Issue by Voxxel** [TLDRMissions/TLDRMissions#600]
**TLDR Version of your idea**
TLDR version: 10.0-005

Please add an ability to exclude specific missions.

**Slightly longer version**
I have full rosters of lvl 60 followers for every covenants and there are some specific missions that equally (too) hard for most of them. Those missions take a helluva lot of time to calculate and the working combinations are usually found within 10.000 and 20.000 tries, but they also fails often even after 30.000 tries. To be frankly doing these specific missions aren't time efficient at all - so if I could, I would easily exclude them without a doubt to make the table game stop wasting my time anymore.

It would be pretty nice to have an input field where we could filter specific missions out via their name or mission IDs, such as:

Dreaded Lukir - Mission ID: 2278
Investigate the Investigators - Mission ID: 2243
etc.

Is it possible somehow to filter out the worst missions making TLDR Missions even more efficient?

**Comment by Voxxel** [comment]
`I would but the addon is going to be dead in a week.`

Oh no, no, why would it be? Will they shut down the SL mission table or going to cripple the rewards in DF?

I though as -similar to older mission tables- SL table remains a still profitable thing to do, good source of chunk of gold and AH mats. I'm still playing Legion table minigame with dozen of my characters for various rewards that can be converted to gold and for other stuff, like Marks of Honor. I'm pretty sure many people are doing it aswell.

**Comment by teelolws** [comment]
They're crippling the rewards in 10.0.2, which I expect to release next week. No more gold.

It doesn't matter anyway. The best solution is to lower that sim limit. By the time it gets to 3000 sims its already tried all the 1/2 follower combinations, which are the most efficient ways. By that point you probably don't want it sending, even if it finds something that works, cause thats a waste of 3 followers for just 10 Anima.

**Comment by Voxxel** [comment]
> They're crippling the rewards in 10.0.2, which I expect to release next week. No more gold.

Damn, that's bad news... Well, I understand and agree with you, it isn't really worth to implement new features then. :(

**Comment by teelolws** [comment]
I would but the addon is going to be dead in a week. I suggest you just lower the sim limit to 2000 or 3000 and let it auto-skip those missions.

2243 already has a very large database of what *cannot* work under the optimum situations. If 2243 takes a long time, its probably trying all the 4-follower combinations and 5-follower combinations and still getting nothing.
